iroh-sync/src/store/fs.rs

```diff
@@ -53,15 +53,24 @@ const NAMESPACES_TABLE: TableDefinition<&[u8; 32], &[u8; 32]> =
 // Value:
 //    (u64, [u8; 32], [u8; 32], u64, [u8; 32])
 //  # (timestamp, signature_namespace, signature_author, len, hash)
+const RECORDS_TABLE: TableDefinition<RecordsId, RecordsValue> = TableDefinition::new("records-1");
+
+// Latest
+// Table
+// Key: ([u8; 32], [u8; 32]) # (NamespaceId, AuthorId)
+// Value: (u64, Vec<u8>) # (Timestamp, Key)
+const LATEST_TABLE: TableDefinition<LatestKey, LatestValue> = TableDefinition::new("latest-1");
```

Comment:
this table will be empty on new versions, resulting in wrong results, we should either do a migration or error on an existing db

Reply:
Good point. I wrote a migration: https://github.com/n0-computer/iroh/pull/1530/commits/88656416ecd8b48d459c37d9dd74012557004740